frontend/src/pages/my-projects/components/create-project-form/libs/constants.ts

```diff
@@ -1,7 +1,21 @@
-import { type ProjectCreateRequestDto } from '~/packages/projects/projects.js';
+import { type SelectOption } from '~/libs/types/types.js';
+import {
+  type ProjectCreateRequestDto,
+  ProjectCategory,
+} from '~/packages/projects/projects.js';
+
+import { projectCategoryToReadable } from './types/types.js';
+
+const OPTIONS: SelectOption<string | number>[] = Object.values(
```

Comment:
Looks like it's not `string | number` - you actually know the type of values passed (`ProjectCategory` vals)